### DISCUSSION TITLE
Can this layout be made available on ZMK?

### DISCUSSION THREAD

**Discussion by harshitgoel96** [manna-harbour/miryoku#49] [Q&A] [answered]
Can this layout be made available on ZMK?

**Comment by manna-harbour** [comment]
Yes! Miryoku ZMK is in development. It currently builds for all ergo splits in ZMK. The keymap can be used now but needs some other work before public release. 

If anyone would like to use it now without redistributing it, just let me know.

**Comment by harshitgoel96** [comment]
> Yes! Miryoku ZMK is in development. It currently builds for all ergo splits in ZMK. The keymap can be used now but needs some other work before public release.
> 
> If anyone would like to use it now without redistributing it, just let me know.

Thanks for the response. In how much time would it be ready for release? Is there any eta at the moment?

I am interested in checking it out.

**Comment by manna-harbour** [comment]
It's waiting on some docs and also a general miryoku reorganisation.  If you have a ZMK board already I can give you access to the repo now. If you happen to be in the Zen GB and are preparing for that I'll make sure it's released before the Zen ships.

**Comment by harshitgoel96** [comment]
I am already using zmk on Lily58 with bluemicro.
Would it be ok if I manually make a keymap myself on zmk ?

**Comment by manna-harbour** [comment]
Yes of course, and there are a few other implementations already. There are also a few people with access to my development version which can be used now. Let's move to discord if you'd like to arrange access.

**Comment by Percentnineteen** [comment]
I'm setting up a Corne on ZMK and was wondering if I could use your dev version as a starting point for my keycap. I was also wondering if you planned to work in bluetooth controls for ZMK given its focus on wireless functionality. Thanks.

**Comment by manna-harbour** [comment]
> I'm setting up a Corne on ZMK and was wondering if I could use your dev version as a starting point for my keycap. I was also wondering if you planned to work in bluetooth controls for ZMK given its focus on wireless functionality. Thanks.

Sure.  Please contact me privately to arrange this.  https://github.com/manna-harbour/manna-harbour/tree/main/contact

Bluetooth controls are on the media layer for the time being.

**Comment by manna-harbour** [comment, marked as answer]
See #50.

**Reply by justalittle-BIT** [reply]
hi! any update on this? is the config keymap file ready? for my 36 crkbd I would like to use the ZMK.

**Reply by manna-harbour** [reply]
It'll be released soon but you're welcome to use it now, just [contact me directly](https://github.com/manna-harbour/manna-harbour/tree/main/contact) to arrange that.
